scripts/toaster-layers-recipes-config.py

```diff
@@ -40,7 +40,7 @@
 
 # Now time to populate custom.xml
 
-for layername in layer_and_recipes:
+for layername in list(layer_and_recipes.keys()):
```

Comment:
@kergoth Yah, you are right. .keys() is redundant. I missed that part, that I am trying to modify dict during iteration. I will update the same in my next version of the patch. Thanks for the valuable feedback.